src/mantine-core/src/components/SegmentedControl/SegmentedControl.styles.ts

```diff
@@ -10,6 +10,7 @@ interface SegmentedControlStyles {
   transitionDuration: number;
   transitionTimingFunction: string;
   size: MantineSize;
+  vertical: boolean;
```

Comment:
This prop should be renamed to `orintation` and have type `'vertical' | 'horizontal'` to be consistent with other components